epxresults/run.py

```diff
@@ -377,6 +377,52 @@ def get_list_variable(
                 count += 1
         return pd.Series(arr, dtype='float')
 
+    def get_table_variable(
+            self,
+            variable: str,
+            sim_day: int = None,
+            return_as_df = False,
```

Comment:
It doesn't look like this is used—perhaps it can be removed?

Reply:
yep. that was left over from dev. We had considered letting the user select which format the data was returned in, a series or python dictionary. We chose to only return a data series to be as consistent as possible over the range of methods.